pkg/worker/worker.go

```diff
@@ -227,7 +227,20 @@ func allOtherContainersCompleted(myNS, myPod, myContainer string) bool {
 
 func handleWaitFile(resultFile, url string, client *http.Client) error {
 	var outfile *os.File
-	var err error
+
+	fi, err := os.Stat(resultFile)
```

Comment:
nit: It's more readable to not abbreviate variable names in most cases. I think "file" is more readable than "fi"

Reply:
In general I agree, but in this case `fi` is not an abbreviation for `file`, its an acronym for `FileInfo` which is the type and I wouldn't want to make it confusing since its actually not a `file` (e.g. you cant read it, it only carries some info about it).  Still reading through other comments but if I fix other stuff up I can rename for more clarity if you like.